### DISCUSSION TITLE
08.01.2024 - Finding Exceptions

### DISCUSSION THREAD

**Discussion by DGrillCoder** [grilldb97/legendary-space-zebra#19] [Show and tell]
Wir haben uns in der Vorlesung folgendes angeschaut:
Conditions:
Equals: a == b
Not Equals: a != b
Less than: a < b
Less than or equal to: a <= b
Greater than: a > b
Greater than or equal to: a >= b

Dictionaries:
KEY:VALUE
nach dem Komma im Code am Besten eine neue Zeile => bessere Lesbarkeit

Cases:
match => zum Vergleichen

Keywords:
AND, OR, NOT
davon haben wir Logic-gates programmiert

Rock, Paper, Scissors programmiert

Code zu Fibonacci kritisch angeschaut und überlegt, was man besser machen kann

nested loop: 1-5 und die entsprechende Zahl soll die Anzahl der Zahl ausgeben: 1, 22, 333,...
2 Prime Checker

RegEx: regular expressions angeschaut

**Comment by SirPatschiii** [comment]
Nice conclusion of the beginning of the week.
I look forward to see more progress in your alarm clock project. There you can maybe make use of some global variables to achieve a better structure and get your program running again.
